apps/web/src/pages/get-started/consts/EchoUseCase.const.tsx

```diff
@@ -0,0 +1,110 @@
+import { StepDescription, StepText } from './shared';
+import { OnboardingUseCase } from './types';
+import { OnboardingUseCasesTabsEnum } from './OnboardingUseCasesTabsEnum';
+import { colors, Copy } from '@novu/design-system';
+import { Group, UnstyledButton, useMantineColorScheme } from '@mantine/core';
+import { useClipboard } from '@mantine/hooks';
+import { useSegment } from '@novu/shared-web';
+
+const COMMAND = 'npx novu-labs@latest echo';
+
+const CodeSnippet = () => {
+  const { copy } = useClipboard();
+  const { colorScheme } = useMantineColorScheme();
+  const isDark = colorScheme === 'dark';
+  const segment = useSegment();
+
+  return (
+    <div
+      style={{
+        background: isDark ? colors.B20 : colors.B98,
+        borderRadius: 12,
+        padding: 16,
+        height: 52,
+        width: '50%',
+        color: isDark ? '#EDEDEF' : colors.B40,
+        marginTop: 8,
+      }}
+    >
+      <Group position="apart">
+        <div>{COMMAND}</div>
+        <UnstyledButton
+          onClick={() => {
+            segment.track(`Copy echo command - [Get Started]`);
+            copy(COMMAND);
+          }}
+        >
+          <Copy width="22" height="22" />
+        </UnstyledButton>
+      </Group>
+    </div>
+  );
+};
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use [panda](https://www.notion.so/novuhq/Using-PandaCSS-at-Novu-7b155517c2ff448ca65527fa7b2c9afe) styling for all new work here. Let me know if you have any questions!‏